pages/common/simplehttpserver.md

```diff
@@ -0,0 +1,24 @@
+# simplehttpserver
+
+> A Go alternative of python's SimpleHTTPServer with additional functionalities.
```

Comment:
```suggestion
> A simple HTTP/S server that support file upload, basic auth, and YAML rules for custom responses.
> A Go alternative to Python's http.server.
```
From the [Python docs](https://docs.python.org/2/library/simplehttpserver.html):

> The SimpleHTTPServer module has been merged into http.server in Python 3.0. The 2to3 tool will automatically adapt imports when converting your sources to 3.0.

Additionally, "alternative of" is grammatically incorrect.

Reply:
> Additionally, "alternative of" is grammatically incorrect.
Yup, "alternative to" is the right grammar